arch/wasm32/wasm.js

```diff
@@ -1479,6 +1479,8 @@ if (!(modules[i].exports.main instanceof Function))
   throw new Error('main() not found');
 
 try {
+  if (modules[i].exports.__wasm_call_ctors instanceof Function)
+    modules[0].exports.__wasm_call_ctors();
```

Comment:
Why check `[i]` and invoke `[0]`?

Reply:
Doh!  I wondered why it didn't fix any failures!  Damn it.

Reply:
lol